src/App.js

```diff
@@ -87,11 +87,15 @@ function App() {
 
         <h5>We stand with everyone fighting on the frontlines</h5>
         <div className="link">
-          <a href="https://github.com/covid19india">covid19india</a>
+          <a href="https://github.com/covid19india"
+          target="_blank"
+          rel="noopener noreferrer">covid19india</a>
```

Comment:
I see that you've also included the "noopener" and "noreferrer", could you add them to the rest of the links below too? Would be much more consistent :)

Reply:
Yes, sure! I will do this. I will get back to you soon.